ftlid/__init__.py

```diff
@@ -5,22 +5,33 @@
 DIRNAME = os.path.dirname(os.path.abspath(__file__))
 
 
-def identify_language(text, model_path=None, with_prob=False, **kwargs):
+def identify_language(text, model=None, model_path=None, with_prob=False,
+                      **kwargs):
     """
     Identifies language the provided text is written in using a pre-trained
-    fastText model. Except for the arguments listed below, anky keyword
-    arguments are fassed further to fastText's predict function.
+    fastText model. Except for the arguments listed below, any keyword
+    arguments are passed further to fastText's predict function.
 
     Args:
         - text: The text whose language is to be identified.
+        - model (optional): Provides an instance of a fastText model used for
+          prediction. Defaults to lid.176.ftz which is included in the package.
+          Maximum one of the model and model_path arguments can be provided.
         - model_path (optional): Provides path to the fastText model used for
           prediction. Defaults to lid.176.ftz which is included in the package.
+          Maximum one of the model and model_path arguments can be provided.
         - with_prob (optional): If True, the function returns both predicted
           languages and their probabilities. Defaults to False.
     """
-    if model_path is None:
-        model_path = DIRNAME + '/lid.176.ftz'
-    model = fasttext.load_model(model_path)
+    if model is None:
+        if model_path is not None:
+            model = load_model(model_path)
+        else:
+            model = DEFAULT_MODEL
+    elif model_path is not None:
+        raise ValueError("Unsure which model to use. Maximum one of the model "
```

Comment:
@swecooo I would probably say "At most one" in here.

Reply:
Thanks for this comment, I was looking terribly for a better formulation but couldn't find one. :)